File: sumPrimes.js
var primes = [2, 3];
var total = 5;
var is_prime = true;

for (var i = 4; primes.length < 10; i++) {
  for (var p in primes) {
    if (i % primes[p] == 0) {
      is_prime = false;
      break;
    } else {
      is_prime = true;
    }
  }
  if (is_prime) {
    primes.push(i);
    total += i;
  }
}

File: sum_primes.py
def print_primes():
    primes = [2, 3]
    i = 4
    while len(primes) < 1000:
        for p in primes:
            if i%p == 0:
                break
            if p == primes[-1]:
                primes.append(i)
        i += 1
    return sum(primes)

sys.stdout.write(str(print_primes()))